src/bin/pgcopydb/catalog.c

```diff
@@ -5952,25 +5947,25 @@ catalog_add_s_namespace(DatabaseCatalog *catalog, SourceSchema *namespace)
 
 
 /*
- * catalog_lookup_s_namespace_by_rlname fetches a  entry from our catalogs.
+ * catalog_lookup_s_namespace_by_nspname fetches a s_namespace entry from our catalogs.
  */
 bool
-catalog_lookup_s_namespace_by_rlname(DatabaseCatalog *catalog,
-									 const char *restoreListName,
-									 SourceSchema *result)
+catalog_lookup_s_namespace_by_nspname(DatabaseCatalog *catalog,
+									  const char *name,
+									  SourceSchema *result)
 {
 	sqlite3 *db = catalog->db;
 
 	if (db == NULL)
 	{
-		log_error("BUG: catalog_lookup_s_namespace_by_rlname: db is NULL");
+		log_error("BUG: catalog_lookup_s_namespace_by_nspname: db is NULL");
 		return false;
 	}
 
 	char *sql =
 		"  select oid, nspname, restore_list_name "
 		"    from s_namespace "
-		"   where restore_list_name = $1 ";
+		"   where nspname is $1 ";
```

Comment:
Why use the `is` operator instead of the equality operator `=`?

Reply:
Ooops fixed.